autoscheduler/scraper/tests/api_tests.py

```diff
@@ -453,6 +460,21 @@ def test_api_course_search_gives_correct_results_csce_space_310_lower(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), expected)
 
+    def test_api_course_search_gives_correct_results_by_title(self):
+        """ Tests that /api/course/search gives the correct response for a search
+            by title
+        """
+        # Arrange
+        expected = {'results': ['CSCE 310 - DATABASE SYSTEMS']}
+        data = {'search': 'Database S', 'term': '201731'}
```

Comment:
Shouldn't it be `Database%20S` to more accurately represent how the frontend will be sending data?

Reply:
Honestly it doesn't actually matter I don't think, as I believe Django Rest Framework automatically replaces the `%20` into spaces when we're handling them. I believe we ran into this when we were originally implementing `/api/course/search`

Reply:
Yeah it does automatically replace them, but since the actual query we send is going to have the %20's in them I figured we might as well change it for authenticity even though it shouldn't matter which one we do